### PR TITLE
fix indentation of openshift manifest

### PR DESCRIPTION
**Pull request by rccrdpccl**
Fix badly indented manifest

### CODE REVIEW

File: openshift/template.yaml [hunk 1]
```diff
@@ -160,16 +160,16 @@ objects:
...
+          volumeMounts:
+          - mountPath: /etc/oauth-proxy/
+            name: oauth-proxy-htpasswd
+            readOnly: true
+        volumes:
+        - name: oauth-proxy-htpasswd
+          secret:
+            defaultMode: 420
+            optional: false
+            secretName: oauth-proxy
```
[the remark this answers is not in the document]
Comment (rccrdpccl) [reply openshift/template.yaml:172]: There is no such secret - are you suggesting to rename it?

[the remark this answers is not in the document]
Comment (michaellevy101) [reply openshift/template.yaml:172]: true for kibana-oauth-application. The Kibana itself is a MO

**Pull request merged**